spec/rails_helper.rb

```diff
@@ -7,6 +7,7 @@
 require File.join(File.dirname(__FILE__), 'support/valid_attribute')
 require File.join(File.dirname(__FILE__), 'support/factory_girl')
 require 'capybara/rspec'
+require 'valid_attribute'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.